### PR TITLE
Help support page

### PR DESCRIPTION
**Pull request by cis-shubham-t**

### CODE REVIEW

**Review by zaSmilingIdiot — COMMENTED** [review]
When resizing the GUI dimensions, the text within each card reflows. When this happens, and the number of lines used by a text blurb in one of the top 2 cards changes (example: the width of the GUI is adjusted, and the description text fort he Wiki item changes from 1 line to 2 lines ), then the card below it changes position, and is no longer 'aligned' with the other card on the same level.

Would it be possible to adjust this so that each card occupies at least the same space, as per the lsiting items on the listing items page in the marketplace?

File: src/app/wallet/help/help.component.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              <p class="desc">For translators</p>
+            </div>
+          </a>
+
+          <a class="item" href="https://particl.wiki" target="_blank">
+            <div class="icon">
+              <mat-icon fontSet="partIcon" fontIcon="part-wikipedia"></mat-icon>
+            </div>
+            <div class="text">
+              <div class="title">Wiki</div>
```
Comment (zaSmilingIdiot) [inline src/app/wallet/help/help.component.html:181]: Is there a reason for the second Wiki entry? It has already been included int he 'Tutorials' section.

Comment (cis-shubham-t) [reply src/app/wallet/help/help.component.html:181]: @AllienWorks could you please address this comment.

**Comment by cis-shubham-t** [comment]
Some important points @AllienWorks mentioned in the comment 
https://github.com/particl/particl-desktop/pull/1122#issuecomment-425894230

as bellow:

We have some outgoing links in the GUI (regarding various "read more" links + all of the links in Help section) - can we whitelist those? Currently the links are not doing anything (worked before), I suspect it's some kind of security measure..

Links to be whitelisted:

https://particl.wiki/ (whole domain)
https://particl.io/ (whole domain or just https://particl.io/user-guides)
https://riot.im/app/#/group/+particl:matrix.org
https://discordapp.com/invite/2tVJaZ9
https://t.me/particlproject
https://particl.news/
https://twitter.com/particlProject
https://www.facebook.com/ParticlProject/
https://www.reddit.com/r/Particl/
https://plus.google.com/+Particl
https://mastodon.social/@particl
https://www.youtube.com/c/Particl
https://bitcointalk.org/index.php?topic=1835782.0
https://github.com/particl
https://www.transifex.com/particl/particl-website/

**Comment by cis-shubham-t** [comment]
> Some important points @AllienWorks mentioned in the comment
> [#1122 (comment)](https://github.com/particl/particl-desktop/pull/1122#issuecomment-425894230)
> 
> as bellow:
> 
> We have some outgoing links in the GUI (regarding various "read more" links + all of the links in Help section) - can we whitelist those? Currently the links are not doing anything (worked before), I suspect it's some kind of security measure..
> 
> Links to be whitelisted:
> 
> https://particl.wiki/ (whole domain)
> https://particl.io/ (whole domain or just https://particl.io/user-guides)
> https://riot.im/app/#/group/+particl:matrix.org
> https://discordapp.com/invite/2tVJaZ9
> https://t.me/particlproject
> https://particl.news/
> https://twitter.com/particlProject
> https://www.facebook.com/ParticlProject/
> https://www.reddit.com/r/Particl/
> https://plus.google.com/+Particl
> https://mastodon.social/@particl
> https://www.youtube.com/c/Particl
> https://bitcointalk.org/index.php?topic=1835782.0
> https://github.com/particl
> https://www.transifex.com/particl/particl-website/

Links open a new tab every time in the browser for me I'm using Linux 16.04.

File: src/app/wallet/help/help.component.html (hunk 1 above)
Comment (AllienWorks) [reply src/app/wallet/help/help.component.html:181]: I understand what you mean. The first link is more like "if you need help, here's Wiki" and the second one is aimed more at contributors, to let them know that Wiki is available as a means to help out. I don't see a problem with it :)

**Comment by AllienWorks** [comment]
> Links open a new tab every time in the browser for me I'm using Linux 16.04.

As I said, it worked for me before, now it isn't. Not sure why, but the links are "dead". I'm on Linux (Manjaro). Any idea @zaSmilingIdiot?

**Comment by zaSmilingIdiot** [comment]
> > Links open a new tab every time in the browser for me I'm using Linux 16.04.
> 
> As I said, it worked for me before, now it isn't. Not sure why, but the links are "dead". I'm on Linux (Manjaro). Any idea @zaSmilingIdiot?

Seems to work for me currently as well: Linux (Fedora 28, chrome/firefox as default browser). This is via the dev startup option though (`yarn start:electron:dev`). @AllienWorks how are you starting up the app during the testing?

**Comment by vikas-cis-zz** [comment]
> > > Links open a new tab every time in the browser for me I'm using Linux 16.04.
> > 
> > 
> > As I said, it worked for me before, now it isn't. Not sure why, but the links are "dead". I'm on Linux (Manjaro). Any idea @zaSmilingIdiot?
> 
> Seems to work for me currently as well: Linux (Fedora 28, chrome/firefox as default browser). This is via the dev startup option though (`yarn start:electron:dev`). @AllienWorks how are you starting up the app during the testing?

yeah it works for me too on mac let me know @AllienWorks how you trying to check 

thanks

File: src/app/wallet/help/help.component.html (hunk 1 above)
Comment (cis-shubham-t) [reply src/app/wallet/help/help.component.html]: @AllienWorks, should I need to remove the duplicate link from the page. Or that option (link) refers to another link in future?

**Comment by zaSmilingIdiot** [comment]
A cursory look around the interwebs seems to suggest that some OS updates (across different distros though) fixed similar problems in other applications. 

Do you maybe have any distro updates that might be pending that you could apply before trying again?

**Comment by AllienWorks** [comment]
I'm also running the dev `yarn start:electron:dev`..

My system is also up to date (I'd even argue I'm ahead as Manjaro is rolling release). Outgoing links worked for me some time ago (maybe like a month). If it's just a case for me, it shouldn't be anything great. So let's keep it as it is, I'll see if I can find the problem.

**Comment by zaSmilingIdiot** [comment]
References JIRA item https://particl.atlassian.net/browse/PD-426

(Creation of  a Help and Support section in the GUI)

**Pull request merged**